### ISSUE TITLE
[A11y][UI]No Visual representation is provided for required field 'Username' 

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8181]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1184475
## Browser Details:
Firefox Version: 79.0 (64 bit)

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450)

## Repro Steps:

    Hit the URL and login with appropriate credentials
    Tab till "Open Profile drop down" and press enter
    Tab till "API Keys" and Press enter
    Tab till "Create button" and press enter
    Tab till "Key Name" editable field
    Check whether a visual representation like asterisk is provided at the required field "Key Name" or not 

## Actual Results:

No Visual representation is provided for required field 'Username'
## Expected Results:
Visual representation like asterisk should be provided at the required fields.
## User Impact:
Users may not understand if there is no visual representation at the required fields.

## Notes:
This appears to be about adding an * by required fields in forms.
